pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -195,16 +195,22 @@ func validateSystemRequirements(mountUtil mount.Interface) (features, error) {
 	return f, nil
 }
 
+// struct to combine a swap options
```

Comment:
nit (non-blocking): a -> the

Reply:
Looks like we dropped this in the next commit. Can we fixup the commits so they are additive?

Reply:
I am not sure how to do that. I can squash all them if thats easier?

Reply:
Yeah, I find other cases where we have changed code from previous commit to next, so could squash if that's easiest.

Reply:
Squashed and pushed up.